utils/metrics/metrics/src/main/java/com/ea/orbit/metrics/MetricsComponent.java

```diff
@@ -36,10 +36,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.inject.Singleton;
+
 import java.lang.reflect.Method;
 import java.util.ArrayList;
 import java.util.List;
 
+@Singleton
```

Comment:
Without this, apparently Container wouldn't start the Component. Is that intended?